src/lib/y2network/widgets/s390_common.rb

```diff
@@ -0,0 +1,322 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "cwm/common_widgets"
+require "cwm/custom_widget"
+
+module Y2Network
+  module Widgets
+    # Widget for setting the s390 device write channel
+    class S390LanCmdTimeout < CWM::InputField
+      # Constructor
+      #
+      # @param settings [Y2Network::InterfaceConfigBuilder]
+      def initialize(settings)
+        textdomain "network"
+
+        @settings = settings
+      end
+
+      # @see CWM::AbstractWidget
+      def label
+        _("&LANCMD Time-Out")
```

Comment:
it is original name? LANCMD looks really strange

Reply:
Yep it is, and used the same label it has in previous dialog.